WhaleSpotting/react-app/src/styles/ReportSighting.scss

```
@@ -0,0 +1,90 @@
+@import "./Constants.scss";
+.report-sighting{
+    font-family: $fontFamily;
+
+    .banner-image{
+        height: 40vh;
+        object-fit: cover;
+        width: 100%;
+    }
+
+    .required{
+        color: $secondaryDark;
+        font-size: $mobileBodyFontSize;
+    }
+
+    .container{
+        margin: auto;
+        max-width: 800px;
+        padding: 4% 5%;
+    }
+
+    .card-component{
+        background: $primaryCard;
+        border-radius: 5px;
+    }
+
+    .title{
+        text-align: center;
+        font-size: $titleFontSize;
+    }
+
+    .sighting-details{
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: space-between;
+        margin-top: 6%;
+        padding: 2% 5% 5% 5%;
+    }
+
+    .input-box{
+        margin: 3% 0;
+        width: 46%;
+    
+        .input-field{
+            height: 85%;
+            width: 100%;
+            padding-left: 10px;
+            border-radius: 5px;
+            transition: all 0.3s ease;
+            border: 1px solid $primaryColour;
+            font-size: $bodyFontSize;
+            
+            &:focus{
+                outline: none;
+                border: 2px solid $primaryDark;
+            } 
+        }
+    }
+
+    .description{
+        width: 200%;
+    }
+        
+    .button{
+        height: 45px;
+        margin: 35px 0;
+    
+        .input-field{
+            height: 100%;
+            width: 100%;
+            color: $textOnPrimary;
+            border: none;
+            font-size: $bodyFontSize;
+            border-radius: 5px;
+            background: $primaryDark; 
+        
+            &:hover{
+                background: $primaryColour;
+            }
+        }
+    }
```

To keep the button styling that I suggested above
```suggestion
    .submit-button{
        height: 45px;
        margin: 35px 0;
        width: 100%;
        color: $textOnPrimary;
        border: none;
        font-size: $bodyFontSize;
        border-radius: 5px;
        background: $primaryDark;

        &:hover{
            background: $primaryColour;
        }
    }
```

---

done